finetune/datasets/__init__.py

```diff
@@ -60,3 +60,22 @@ def generic_download(url, text_column, target_column, filename, save=True, task_
         new_df.to_csv(save_path, index=False)
 
     return new_df
+
+
+def comparison_download(url, text_column1, text_column2, target_column, filename, save=True, task_type='Similarity', data_directory='Data'):
```

Comment:
Can combine with the other download_fn if you can tell me what interface youd like.

Reply:
could standardize on input_columns